testing/dbtest/docker/Makefile

```diff
@@ -49,4 +59,12 @@ clean:
 	docker stop $(TEST_CONTAINER_NAME) || true
 	docker rm -v $(TEST_CONTAINER_NAME) || true
 
-.PHONY: all docker-build database-up ${docker-buildxs} clean
+generate-database-dumps:
```

Comment:
can you add a target to the top level Makefile that calls this target?

We do this for other targets in this makefile like [test-database-up](https://github.com/hashicorp/boundary/blob/a1cf3234ed088a738b53997eeee0eeb8883b9ef7/Makefile#L209-L211)

Reply:
Fixed